docker/minio/env.template

```diff
@@ -4,5 +4,8 @@ MINIO_ROOT_USER=test_key
 # AWS secret key
 MINIO_ROOT_PASSWORD=test_secret
 
+# MC configuration
```

Comment:
MC is indeed [MinIO Client](https://min.io/docs/minio/linux/reference/minio-mc.html) but since they use the name `mc` throughout the docs I wasn't sure if that is an acronym.